Tests/FBLPromisesTests/FBLPromise+WrapTests.m

```diff
@@ -220,13 +220,13 @@ - (void)testPromiseWrapErrorOrObjectCompletionRejectsOnValueAndErrorReturned {
 
 - (void)testPromiseWrap2ObjectsOrErrorCompletionFulfillsOnValueReturned {
   // Arrange.
-  NSArray<NSNumber *> *expectedValues = @[ @42, @13 ];
+  NSArray<NSObject *> *expectedValues = @[ @42, [NSNull null] ];
```

Comment:
Just skip the generic arg, ie. `NSArray *expectedObjects`, which is a synonym for `NSArray<id>`

Reply:
That makes sense.
I have updated the above.